app/models/form_handler.rb

```diff
@@ -41,6 +43,23 @@ def lettings_forms
         forms[form_to_set] = form if forms[form_to_set].blank?
       end
     end
+
+    lettings_sections = [
+      Form::Lettings::Sections::TenancyAndProperty,
+      Form::Lettings::Sections::Household,
+      Form::Lettings::Sections::RentAndCharges,
+    ]
+
+    if forms["previous_lettings"].blank? && current_collection_start_year >= 2022
+      forms["previous_lettings"] = Form.new(nil, current_collection_start_year - 1, lettings_sections, "lettings")
+    end
+    if forms["current_lettings"].blank? && current_collection_start_year >= 2022
+      forms["current_lettings"] = Form.new(nil, current_collection_start_year, lettings_sections, "lettings")
+    end
+    if forms["next_lettings"].blank? && current_collection_start_year >= 2022
+      forms["next_lettings"] = Form.new(nil, current_collection_start_year + 1, lettings_sections, "lettings")
+    end
```

Comment:
Why do we need `current_collection_start_year >= 2022` conditions?

Reply:
Cause if it's 2021 Form.new will try to create previous_lettings for 20/21 from a JSON that we don't have. Needs a refactor tho

Reply:
Yes, so sorry, I didn't realise this was in draft. Safely ignore my comments until you're ready!